crates/next-core/src/next_server/context.rs

```diff
@@ -1010,7 +1010,10 @@ pub async fn get_server_chunking_context_with_client_assets(
 
     if next_mode.is_development() {
         builder = builder.use_file_source_map_uris();
+    } else {
+        builder = builder.ecmascript_chunking_config(ChunkingConfig {})
     }
```

Comment:
We can leave this for now, but it's somewhat surprising that `.ecmascript_chunking_config(ChunkingConfig {})` alone enables production chunking.